### PR TITLE
Merges feature/tools update branch into devel

### PR DESCRIPTION
**Pull request by exoticDFT**
- Adds all of the startup scripts for the drone and reorganizes them into two folders

  1. Drone: Which is a replica of the OS file system structure, so that the folders within can just be copied in one simple command.
  2. scripts: Which holds some secondary scripts useful for speeding up our experiments.

- Adds a README file to the Tools directory to help explain the various tools.

### CODE REVIEW

**Comment by exoticDFT** [comment]
@lowjunen @kayn329  I would definitely do a relatively thorough read through of the README I have here. I tried to add everything I believe we needed to do on a new drone, but I'm sure some of the descriptions could be improved. Just let me know in the review and I will fix whatever.

File: Tools/Drone/etc/hosts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+127.0.0.1       localhost
+127.0.1.1       drone4 drone4.local
```
Comment (lowjunen) [inline Tools/Drone/etc/hosts:2]: does $(hostname) not work here?

File: Tools/README.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  filesystem.
+- **scripts**: Some random scripts that can be useful for interacting with the
+  drones.
+
+Installing the Drone files
+==========================
+
+The Drone directory contains several files needed to setup the drone when
+powered on. These files contain system level services and some configurations
+needed for the drone to run correctly. If there files under this directory have
```
Comment (lowjunen) [inline Tools/README.rst]: minor typo.

**Review by lowjunen — COMMENTED** [review]
just the two comments (hostname and typo).

File: Tools/README.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+- `sudo visudo`
+  
+  - Add the line `carl ALL=(ALL) NOPASSWD: ALL` after the `%sudo ALL=(ALL:ALL) ALL`
+  
+- `sudo rsync -rvulD ~/StanfordMSL/TrajBridge-PX4/Tools/Drone/* /.`
+
+  - You can add `-n` to the end of this command to verify the copy.
+
+- 'sudo systemctl daemon-reload`
```
Comment (kayn329) [inline Tools/README.rst]: ' instead of `

File: Tools/README.rst [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+The first updates the sudoer file to include "carl" as a sudoer without the need
+of a password. This is needed for running some of the startup scripts.
+
+This `rsync` command will copy all of the files under the "Drone" directory to
+their appropriate locations on the drone's filesystem. This command will need to
+be run anytime the files inside the Drone directory are updated.
+
+The `systemctl` commands all interact with the operating system. Specifically,
+reloading and enabling our scripts to run at startup. The `daemon-reload` needs
+to be run anytime the service files have changed. The next three updates the
```
Comment (kayn329) [inline Tools/README.rst]: The next three update

File: Tools/README.rst (hunk 3 above)
Comment (kayn329) [inline Tools/README.rst]: ' instead of `

**Review by kayn329 — COMMENTED** [review]
Just two minor typos. I think using (`) for lines of code isn't working, might need to use something else for .rst files?

File: Tools/Drone/etc/hosts (hunk 1 above)
Comment (exoticDFT) [reply Tools/Drone/etc/hosts:2]: I sadly could not get something like this to work. Let me see if I can figure out a way, as this is definitely the better way to handle the file, if it is possible.

File: Tools/README.rst (hunk 3 above)
Comment (exoticDFT) [reply Tools/README.rst]: Good catch. I will fix it!

File: Tools/README.rst (hunk 4 above)
Comment (exoticDFT) [reply Tools/README.rst]: Good catch

**Comment by exoticDFT** [comment]
> Just two minor typos. I think using (`) for lines of code isn't working, might need to use something else for .rst files?

You are very right. I forgot I have to use the "directives" here. Specifically, :code:`this is a command`.

**Comment by lowjunen** [comment]
noticed this is still open. should we throw in the switching feature (aether vs relay) as updates to this PR or open a new one @exoticDFT ?

**Pull request closed without merging**